backup.c

```diff
@@ -1383,6 +1381,18 @@ dirExists(const char *path)
 		return true;
 }
 
+/*
+ * Return true if standby.signal file exists and
+ * store the file path to the "path"
+ */
+bool
+get_standby_signal_filepath(char *path, size_t size)
```

Comment:
For the newly added get_standby_signal_filepath function, it is called when backup and restore.
However, the function is defined in backup.c.
So how about moving to another source code where the function is defined?
For example, I think it would be better to define it in data.c or 
What do you think of this opinion?

Reply:
OK, will fix.

Reply:
Fixed not only get_standby_signal_filepath() but also fileExists() because both used in backup.c and restore.c.